protostar/utils/data_transformer_facade.py

```diff
@@ -70,6 +72,26 @@ def _find_abi_item(contract_abi: AbiType, name: str) -> Dict:
                 return item
         raise AbiItemNotFoundException(f"Couldn't find '{name}' ABI")
 
+    @classmethod
```

Comment:
I am not sure if it belongs to `DataTransformerFacade`, maybe we should keep it as a plain helper function?